SpotifyAPI/Local/SpotifyLocalAPI.cs

```diff
@@ -139,11 +140,18 @@ public Boolean Connect()
             return _rh.Init();
         }
 
-        public StatusResponse GetStatus()
+        /// <summary>
+        /// Gets the Spotify status
+        /// </summary>
+        /// <returns></returns>
+        public async Task<StatusResponse> GetStatus()
```

Comment:
Seperate async methods would be totally fine. Especially if you want to be careful for other people scripts.

But the void ElapsedTick(object sender, ElapsedEventArgs e) must also be async in this case.

If its ok you, we need a ElapsedTick and a ElapsedTickAsync also.

Not Blocking the UI thread is mandatory